crates/lovely-win/src/lib.rs

```diff
@@ -47,27 +51,30 @@ unsafe extern "system" fn DllMain(_: HINSTANCE, reason: u32, _: *const c_void) -
         );
     }));
 
-    let args = env::args().collect_vec();
+    let args = env::args().collect::<Vec<_>>();
```

Comment:
I assume you changed this?

Reply:
Yes

Reply:
Replace with `.collect_vec` to keep it consistent with the rest of the codebase. If not applicable then ignore.

Reply:
Okay, done.